tests/phpunit/tests/rest-api/rest-schema-setup.php

```diff
@@ -372,6 +372,9 @@ public function test_build_wp_api_client_fixtures() {
 			$mocked_responses .= "\nmockedApiResponse." . $route['name'] . ' = '
 				. json_encode( $fixture, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES )
 				. ";\n";
+		}
+
+		// Only generate API client fixtures in single site and when required JSON_* constants are supported.
```

Comment:
This block needs to be removed too, looks like probably an invalid conflict resolution.